### PR TITLE
Fix issue where service is not shutdown if interrupted while interactive prompt is initialising. Closes #1004

### LINKED ISSUE #1004: In interactive mode, if ctrl+d is hit before initialisation is complete, the database is not shut down

### PR DESCRIPTION
**Pull request by binaek**

### CODE REVIEW

File: interactive/interactive_client.go [hunk 1]
```diff
@@ -99,10 +96,9 @@ func (c *InteractiveClient) InteractivePrompt() {
 		// close up the SIGINT channel so that the receiver goroutine can quit
 		signal.Stop(interruptSignalChannel)
 		close(interruptSignalChannel)
-		// close the underlying client if it exists
-		if client := c.client(); client != nil {
-			client.Close()
-		}
+
+		// init data cleanup
```
Comment (kaidaguerre) [inline interactive/interactive_client.go:100]: need a better comment here - this is important - this cleanup shuts the service
```
// cleanup the init data to ensure any services we started are stopped
```

File: interactive/interactive_client_init.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	"github.com/turbot/steampipe/db/db_common"
+	"github.com/turbot/steampipe/workspace"
```
Comment (kaidaguerre) [inline interactive/interactive_client_init.go:11]: remove gap

**Review by kaidaguerre — CHANGES_REQUESTED** [review]
looks good - minor tweaks

File: interactive/interactive_client.go (hunk 1 above)
Comment (binaek) [reply interactive/interactive_client.go]: 👍

File: interactive/interactive_client_init.go (hunk 2 above)
Comment (binaek) [reply interactive/interactive_client_init.go]: done

**Pull request merged**